tests/snapshots/TestCLICommands_atmos_about_--help.stdout.golden

```diff
@@ -13,21 +13,28 @@ Flags:
 
 Global Flags:
 
-        --logs-file string          The file to write Atmos logs to. Logs can be
-                                    written to any file or any standard file
-                                    descriptor, including /dev/stdout,
-                                    /dev/stderr and /dev/null (default
-                                    /dev/stderr)
-
-        --logs-level string         Logs level. Supported log levels are Trace,
-                                    Debug, Info, Warning, Off. If the log level
-                                    is set to Off, Atmos will not log any
-                                    messages (default Info)
-
-        --redirect-stderr string    File descriptor to redirect stderr to.
-                                    Errors can be redirected to any file or any
-                                    standard file descriptor (including
-                                    /dev/null)
+        --base-path string           Base path for Atmos project
+
+        --config stringSlice         Paths to configuration file (default [])
```

Comment:
Also we don't specify how to pass the list or if the flag can be repeated
Multiple times